scheduler/src/cook/mesos/api.clj

```diff
@@ -947,7 +950,8 @@
         reason (reason/instance-entity->reason-entity db instance)
         exit-code (:instance/exit-code instance)
         progress (:instance/progress instance)
-        progress-message (:instance/progress-message instance)]
+        progress-message (:instance/progress-message instance)
+        file-url (plugins/file-url file-plugin/plugin instance)]
```

Comment:
Why not put this in the cond-> section, and omit it if the result is nil?

Reply:
What do you mean? `file-url` is already in the `cond->`. If you mean removing the variable assignment, I think it's more clear and matches the other fields in `cond->` which all have variables defined.